git/README.md

```diff
@@ -7,24 +7,53 @@ OCA maintainer quality tools.
 
 For using it, you can choose from two options:
 
-1. Copy files to local git repository into .git/hooks, and you will enable the
+1. Create symbolic link of files to local git repository into .git/hooks, and you will enable the
    checks for that git repository.
 
-2. Copy files to git template directory. In Ubuntu, it's located in:
+   ```bash
+   git clone git@github.com:OCA/maintainer-quality-tools.git ${HOME}/lint-hook
+   ln -sf ${HOME}/lint-hook/git/* {YOUR_PROJECT}/.git/hooks/.
+   ```
+
+2. Create symbolic link of files to git template directory.
+
+  In Ubuntu, it's located in:
+
+     /usr/share/git-core/templates/hooks/
+
+
+  In OS X, it's located in:
+
+    Installed from MAC-port
+
+        /opt/local/share/git-core/templates/hooks/
 
-   /usr/share/git-core/templates/hooks/
 
 Then, making `git init` inside a git repository, these files will be copied,
 and you will have these checks available for it.
 
-**IMPORTANT** Don't forget to install/update flake8, pep8 and pylint modules to
+   ```bash
+   git clone git@github.com:OCA/maintainer-quality-tools.git ${HOME}/lint-hook
+   sudo ln -sf ${HOME}/lint-hook/* /usr/share/git-core/templates/hooks/.
+   cd {YOUR_PROJECT}
+   git init .
+   ```
+
+**IMPORTANT** Don't forget to install/update flake8, pep8 and pylint-odoo modules to
 have the same checks as Travis. Don't use distribution packages, as they are 
 outdated. Use pip ones:
-```
-sudo pip install flake8 --upgrade
-sudo pip install pep8 --upgrade
-sudo pip install pylint --upgrade
+
+```bash
+# pip install --upgrade flake8 pep8
+$ git clone https://github.com/OCA/pylint-odoo
+$ cd pylint-odoo
+# ./install.sh
```

Comment:
Check by yourself the change: https://github.com/OCA/maintainer-quality-tools/commit/a55cdc5e115917973730dea75bfc1adef7bf7a04